api/src/main/scala/hmda/api/http/institutions/SubmissionPaths.scala

```diff
@@ -141,4 +141,43 @@ trait SubmissionPaths
       }
 
     }
+
+  def submissionSingleEditPath(institutionId: String) =
+    path("filings" / Segment / "submissions" / IntNumber / Segment) { (period, submissionId, editType) =>
+      val path = s"institutions/$institutionId/filings/$period/submissions/$submissionId/$editType"
+      extractExecutionContext { executor =>
+        implicit val ec: ExecutionContext = executor
+        val supervisor = system.actorSelection("/user/supervisor")
+        val fHmdaFileValidator = (supervisor ? FindProcessingActor(HmdaFileValidator.name, SubmissionId(institutionId, period, submissionId))).mapTo[ActorRef]
+
+        val fValidationState = for {
+          s <- fHmdaFileValidator
+          xs <- (s ? GetState).mapTo[HmdaFileValidationState]
+        } yield xs
+
+        val fSingleEdits = fValidationState.map { editChecks =>
+          editType match {
+            case "syntactical" =>
+              val s = validationErrorsToEditResults(editChecks.syntactical, Syntactical)
+              SummaryEditResults(s, EditResults.empty, EditResults.empty, EditResults.empty)
```

Comment:
Is this the correct format for returning a single type of error?  Or should I simply return an `EditResults` type, rather than wrapping it in `SummaryEditResults`?